### PR TITLE
Puneet/update to use android x and u7

### PR DESCRIPTION
**Pull request by puneet-pdx**

### CODE REVIEW

**Comment by puneet-pdx** [comment]
@gunt0001 can you please review these changes. The changes are for the app to use runtime Update 7 and Androidx libraries.

File: maps-app/src/main/res/layout/maps_app_activity.xml [hunk 1]
```diff
@@ -22,4 +22,4 @@
         android:divider="@color/colorDivider"
         android:dividerHeight="1px"/>
 
-</android.support.v4.widget.DrawerLayout>
\ No newline at end of file
+</androidx.drawerlayout.widget.DrawerLayout>
```
Comment (changanxian) [inline maps-app/src/main/res/layout/maps_app_activity.xml]: add a blank line

Comment (puneet-pdx) [reply maps-app/src/main/res/layout/maps_app_activity.xml]: updated

**Comment by puneet-pdx** [comment]
@eri9600 can you review.

File: build.gradle [hunk 2]
```diff
@@ -24,12 +24,16 @@ ext {
     compileSdkVersion = 29
 
     // App dependencies
-    supportLibraryVersion = '28.0.0'
-    runtimeVersion = '100.6.0'
-    butterknifeVersion = '8.8.1'
+    appcompatLibraryVersion = '1.1.0'
```
Comment (eri9600) [inline build.gradle]: I would suggest moving all of your versions to a gradle file called `version.gradle` like the samples team did [here](https://github.com/Esri/arcgis-runtime-samples-android/blob/master/version.gradle) just to keep everything uniform.

**Comment by puneet-pdx** [comment]
@esreli do you wanna look at the new release I added to RELEASE.md file before I go ahead and merge this ?

**Comment by esreli** [comment]
@puneet-pdx sure. LGTM.

**Pull request merged**